change/@fluentui-react-10ad3362-2fb7-408f-8231-72299a0b73fa.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Can you delete this file? I'm not sure why there are two, and this one doesn't have a message.

Reply:
okay sure. I'll do it